src/GoogleAdWordsEventForwarder.js

```diff
@@ -28,12 +28,39 @@
         },
         ENHANCED_CONVERSION_DATA = "GoogleAds.ECData";
 
+    // Declares valid Google consent values
+    var googleConsentValues = {
+        // Server Integration uses 'Unspecified' as a value when the setting is 'not set'.
+        // However, this is not used by Google's Web SDK. We are referencing it here as a comment 
+        // as a record of this distinction and for posterity.
+        // If Google ever adds this for web, the line can just be uncommented to support this.
+        //
+        // Docs:
+        // Web: https://developers.google.com/tag-platform/gtagjs/reference#consent
+        // S2S: https://developers.google.com/google-ads/api/reference/rpc/v15/ConsentStatusEnum.ConsentStatus
+        // 
+        // Unspecified: 'unspecified',
+        Denied: 'denied',
+        Granted: 'granted',
+    };
+
+    // Declares list of valid Google Consent Properties
+    var googleConsentProperties = [
+        'ad_storage',
+        'ad_user_data',
+        'ad_personalization',
+        'analytics_storage',
+    ];
+
     var constructor = function () {
         var self = this,
             isInitialized = false,
             forwarderSettings,
             labels,
             customAttributeMappings,
+            consentMappings,
```

Comment:
Let's make this an empty array.  This is a safety check. I think if the code is pushed as is, errors would be thrown when it expects `consentMapping` to be an array.

I think if you update an old test with a consent in the event that is passed, you'll run into this error.  Specifically it would happen on inside of `generateConsentStatePayloadFromMappings`. If `mappings` remains `undefined`, but there is consent on the event, then `undefined.length` in line 392 would throw an error.

```suggestion
            consentMappings = [],
```

Reply:
I think so long as all of these new variables have default values we are good. However I would run a manual test where you are using this built code, but not passing in any new configs from the DB.  Set consent, then try to log an event. Does anything else error out?